### ISSUE TITLE
make note of REPEC

### ISSUE THREAD

**Issue by larsvilhuber** [OpenScienceMOOC/Module-6-Open-Access-to-Research-Papers#22]
You fail to mention in your resources http://repec.org/, one of the oldest open indexes to grey literature (mostly in economics). It is not centralized, but has been around since the last 1990s, and has links between published (paywalled) articles and their corresponding working papers.

**Comment by tosteiner** [comment]
@larsvilhuber excellent resource, thank you for bringing this to our attention! Just briefly scrolled through the descriptions, and was wondering: Is it true that all content referenced by REPEC is open access in one way or the other? I just tried some of the links, and those I sampled didn't mention open licensing or other attribution of any sort ...

**Comment by larsvilhuber** [comment]
No, not all content is open access. However, I believe repec itself does not capture license, only whether a document is free to download or not (see the redif metadata schema it uses). That's at least partially due to the fact that the metadata schema stems from the early 2000s, or late 1990s, when there was less concern about the specific license (and CC BY was a glimmer in the future). I have a reference somewhere about an early history of repec and more details, can provide later.

--
Lars Vilhuber
on mobile device

________________________________

**Comment by larsvilhuber** [comment]
Here's an excerpt from a working paper (sorry, not yet public) that I wrote last year:

> Since grey literature at the time was not
cataloged or indexed by most bibliographic indexes, a distinct effort to identify both working
papers and the novel electronic versions grew from modest beginnings in 1992 at Université de
Montréal and elsewhere into what is today known as the Research Papers in Economics
(RePEc) network, a “collaborative effort by hundreds of volunteers in 99 countries” (25–27). The
initial index was split into electronic (WoPEc) (28) and printed working papers (BibEc) (26, 29),
testimony to the prevalence of the exchange of scientific research in semi-organized ways.
Economists had, in fact, access to a central repository for submitting working papers, based on
the arXiv system, but it seems to not have been very popular, in contrast to the decentralized
working paper archives (28). In 1997, BibEc counted 34,000 working papers from 368 working
paper series (30). RePEc today has data from around 4,600 working paper series and claims
about 2.5 million full-text (free) research items, provided in a decentralized fashion by about
2,000 archives (31). These items not only include traditional research papers, but also, since
1994, computer code (32–34). Although still cataloging mostly grey literature, RePEc
bibliographic metadata is, in fact, indexed by all major bibliographic indexes.

References for the above:

26. Krichel T, Zimmermann C (2009) The Economics of Open Bibliographic Data Provision.
Economic Analysis and Policy 39(1):143–152.
27. Bátiz-Lazo B, Krichel T (2012) A brief business history of an on-line distribution system for
academic research called NEP, 1998-2010. Journal of Management History 18(4):445–
468.
28. Krichel T (1997) WoPEc: Electronic Working Papers in Economics Services. Ariadne (8).
Available at: http://www.ariadne.ac.uk/issue8/wopec [Accessed July 19, 2018].
29. Cruz JMB, Krichel T (2000) Cataloging Economics Preprints. Journal of Internet Cataloging
3(2–3):227–241.
30. BibEc main page (1997) Available at:
http://web.archive.org/web/19971211044921/http://netec.mcc.ac.uk:80/BibEc.html
[Accessed July 19, 2018].
31. IDEAS/RePEc Available at: https://ideas.repec.org/ [Accessed July 19, 2018].
26
32. Economics Software | IDEAS/RePEc Available at: https://ideas.repec.org/i/c.html [Accessed
July 19, 2018].
33. Eddelbüttel D (1997) A Code Archive for Economics and Econometrics. Computational
Economics 10(4). Available at:
http://web.archive.org/web/19980515055648/http://netec.mcc.ac.uk:80/~adnetec/CodEc/c
e97.pdf [Accessed July 19, 2018].
34. CodEc - Programs for Economics and Econometrics (1998) Available at:
http://web.archive.org/web/19980121224535/http://netec.mcc.ac.uk:80/CodEc.html
[Accessed July 19, 2018].

**Comment by larsvilhuber** [comment]
ReDIF documentation: http://openlib.org/acmes/root/docu/redif_1.html

The only encoding of anything resembling a license is in the "File-restriction:" field:

> Restriction: A restriction on the retrieval of a file. If the file can be retrieved by anybody with Internet access without any preliminary payment or registration, you should not use that field at all. As soon as there is a restriction on the file many services will be assuming that there is no public access to the file. Some user services may wish to display only non-restricted files and your file will not appear there. If all files of a document are restricted, please use the field Restriction: in the template of the resource, rather than repeating it for each file of the resource.

All ReDIF files are on a public server, so it should be feasible to parse them for any additional info.

**Comment by Protohedgehog** [comment]
Excellent, thank you, I will try and work this in today! :)
